layouts/sessions/list.html

```diff
@@ -124,4 +124,6 @@ <h3 class="heading heading--3 session-day-section__hour-heading">
         "ctaUrl" .Site.Params.themes.event.callToAction.other.url
         )
     }}
+
+    {{- partial "sections/quick-navigation.html" (dict "menu" .Site.Menus.main) -}}
```

Comment:
Please let confirm Daria that this menu should only be displayed on these three pages and not on every page (like all other menus). If it should be on all pages, we can move it to the `baseof` template (like all other menus).

Reply:
Discussed on the call, added to each page